pierone/cli.py

```diff
@@ -237,6 +218,27 @@ def cves(config, team, artifact, tag, url, output):
     print('\x1b[1;33m!! THIS FUNCTIONALITY IS DEPRECATED !!\x1b[0m', file=sys.stderr)
 
 
+@cli.command()
+@click.argument('incident', callback=validate_incident_id)
+@click.argument('team', callback=validate_team)
+@click.argument('artifact')
+@click.argument('tag')
+@url_option
+@click.pass_obj
+def mark_production_ready(config, incident, team, artifact, tag, url):
+    """
+    Manually mark image as production ready.
+    """
+    pierone_url = set_pierone_url(config, url)
+    registry = get_registry(pierone_url)
+    image = DockerImage(registry, team, artifact, tag)
+    api = PierOne(pierone_url)
+    api.mark_production_ready(image, incident)
+    print("🧙 Marked {} as `production_ready` due to incident {}.".format(
```

Comment:
Consult the sorcerer emoji with @hjacobs 😁

Reply:
🧙 demoted to easter egg for automata and torch.